src/components/app/App.test.tsx

```diff
@@ -1,8 +1,8 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
-import App from './App';
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import App from "./App";
 
-test('renders learn react link', () => {
+test("renders learn react link", () => {
```

Comment:
Are you really need this test? 
This one related to the tutorial, so you probably need to remove it

Reply:
fixed